src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,65 @@
+package storage;
+
+import tasks.*;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Storage {
+    private File file;
+    public Storage(String filePath) {
+        this.file = new File(filePath);
+        try {
+            if (!file.exists()) {
+                file.getParentFile().mkdirs();
+                file.createNewFile();
+            }
+        } catch (IOException e) {
+                System.out.println(e.getMessage());
+        }
+    }
+
+    public void save(TaskList tasks) {
+        try {
+            tasks.writeToFile(this.file);
+        } catch (IOException ioException) {
+            System.out.println(ioException.getMessage());
+        }
+    }
+
+    public void load(TaskList tasks) {
+        try {
+            Scanner sc = new Scanner(this.file);
+            while(sc.hasNext()) {
+                String s = sc.nextLine();
+                String[] task = s.split(" \\| ");
+                Task t;
+                switch(task[0]) {
+                case "T":
```

Comment:
I did not think of using the first letter as switch cases. Will take as inspiration!!